src/core/qgsstringutils.h

```diff
@@ -189,6 +189,7 @@ class CORE_EXPORT QgsStringUtils
       AllLowercase = QFont::AllLowercase,  //!< Convert all characters to lowercase
       ForceFirstLetterToCapital = QFont::Capitalize, //!< Convert just the first letter of each word to uppercase, leave the rest untouched
       TitleCase = QFont::Capitalize + 1000, //!< Simple title case conversion - does not fully grammatically parse the text and uses simple rules only. Note that this method does not convert any characters to lowercase, it only uppercases required letters. Callers must ensure that input strings are already lowercased.
+      CamelCase = QFont::Capitalize + 1001, //!< Convert the string to camel case. Note that this method does not unaccent characters.
```

Comment:
To avoid holy wars, maybe safer to name this CameCaseWithInitialCap ?

Reply:
Ah true !